frontend/src/components/PostAddingForm/PostAddingForm.jsx

```diff
@@ -110,11 +137,31 @@ const PostAddingForm = ({ formSubmit }) => {
       <ContentInput
         type="text"
         placeholder="Enter announcement message..."
-        {...register('content', { required: true })}
+        {...register('content', {
+          required: "This field can't be empty",
+          pattern: {
+            value: /.*[^\s].*/,
+            message: "This field can't contain only whitespaces",
+          },
+          minLength: {
+            value: 3,
+            message: 'Your message should be at least 3 characters long',
+          },
+          maxLength: {
+            value: 300,
+            message: "Your message shouldn't exceed 300 characters",
+          },
+        })}
       />
+
       <div>
         <Button type="submit">Add</Button>
       </div>
+      <ErrorMessage
+        errors={errors}
+        name="content"
+        render={({ message }) => <ErrorText>{message}</ErrorText>}
+      />
```

Comment:
Jak to wrzucisz do jakiegoś kontenera, nadasz mu display flex, justify-content: space-between i flex-direction:  row-reverse, to ta wiadomość będzie się pojawiać w jednej linii, zamiast pod przyciskiem submit :D

Reply:
To to w ogole tez zauwazylem dopiero teraz, bo bylo normalnie wczesniej a tu sie okazalo ze wkleilem to pod submita a powinno byc nad nim i juz wszystko smiga nie trzeba nic robic 👯